tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -161,6 +161,18 @@ def test_absolute(self):
     def test_absolute_negative(self):
         self.check_unary_negative('absolute')
 
+    @testing.for_all_dtypes(no_complex=True)
+    @testing.numpy_cupy_array_equal()
```

Comment:
```suggestion
    @testing.numpy_cupy_allclose(atol=1e-5)
```
to align with other tests in the file.